test/servinge2e/servicemesh_test.go

```diff
@@ -0,0 +1,155 @@
+package servinge2e
+
+import (
+	"github.com/openshift-knative/serverless-operator/test"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgTest "knative.dev/pkg/test"
+	serving "knative.dev/serving/pkg/apis/serving/v1"
+	"testing"
+)
+
+const (
+	// A test namespace that is part of the ServiceMesh (setup by "make install-mesh")
+	serviceMeshTestNamespaceName = "default"
+	serviceMeshTestImage = "gcr.io/knative-samples/helloworld-go"
+)
+
+func ksvcWithAnnotations(name, namespace, image string, annotations map[string]string) *serving.Service {
```

Comment:
It would be better to modify https://github.com/openshift-knative/serverless-operator/blob/master/test/service.go#L19 and use it?